synapse/cortex.py

```diff
@@ -1618,15 +1618,14 @@ async def getCoreQueue(self, iden):
         return self.multiqueue.status(iden)
```

Comment:
Is there a way to make this not throw an exception? the `get*` functions should return None if the thing isn't found. `req*` functions should throw.